aioresponses/core.py

```diff
@@ -471,6 +471,10 @@ async def match(
 
             if matcher.repeat is False:
                 del self._matches[key]
+            elif matcher.repeat == 2:
```

Comment:
why equal to 2?

Reply:
Because I meant `repeat=1` to have the same effect as `repeat=False`. That is, I interpret the number of repetitions to include the first call. I guess an alternative would be to define the number of repetitions as the number after the first call.

That is, for `repeat=5`, either:
- respond 5 times with foo
- respond with foo, and also do that the next 5 times (so 6 in total)

The former seems more intuitive to me.

Reply:
The `repeat=1` case was actually broken. I fixed it and added a test. I also elaborated on this matter in the main README.